docker/rails/docker-compose.yml

```diff
@@ -9,6 +9,7 @@ x-base-app-config: &base-app-config
     - ~/.gitconfig:/root/.gitconfig:ro
   environment:
     - WAIT_HOSTS=db:5432, cache:6379
+    - COMPOSE_PROJECT_NAME=rails
```

Comment:
This might cause problems because it namespaces a project, which will limit a developer's ability to run concurrent rails projects.

Reply:
Right, from my experience i've never had to run concurrent rails projects (since if we try to run the docker-compose in multiple repos, we suffer from port collision in the database anyway) and I find myself having to teardown existing containers in order to restart new ones. That's why if we just run the containers under one common namespace, we can test on the shared environment in any Rails repo

Reply:
WDYT?

Reply:
Oh this is an artifact of our partial rollout.; I simply treated them as individual projects, but the end-goal would be to have the ability to run the entire Set stack concurrently. 

As it relates to shared dependencies, like databases, we have a few options:
* we can assign a random port by assigning "0" for the host (easy)
* have the shared resource separate from the individual services/repositories (more difficult to coordinate)

Reply:
To move this forward PR, we should consider breaking apart the `entrypoint.sh` changes from the `docker-compose` changes, the former seems immediately pretty useful.